src/pages/updates/whats-new.mdx

```diff
@@ -0,0 +1,65 @@
+---
+
+import { PageDescription, AnchorLinks, AnchorLink } from 'gatsby-theme-carbon';
+
+<PageDescription>
+
+The IBM Design Language is a constantly evolving product. The Digital Design Language is being developed in agile sprint cycles. This page lists all major updates, by release date. Check back here to learn what is new in the latest release and what is coming soon.
+
+</PageDescription>
+
+## Pictogram library
+
+We have a new library with over 600 (confirm number) pictograms available to use and even more to come in the next release. Along with the library comes an available adobe illustrator master file with all the pictograms organized by category. This library and master file will serve as the single source of truth representing the latest and approved pictograms. If you need pictograms or want to contribute to the IBM master file/library please follow the guidance outlines [here](https://www.ibm.com/design/language/elements/pictograms/contribute) and checkout the [library](https://www.ibm.com/design/language/elements/pictogram-library/).
+
+<Row>
+<Column colMd={6} colLg={8}>
+
+![Exporting stroke app icons: step 3](/images/pictos.svg)
+
+</Column>
+</Row>
+
+Last updated on 9 October, 2019
+
+## Search and filtering
+
+This cycle we released a major enhancement to our UI Icon and Pictogram libraries by adding search and category filtering making it much easier to find what you’re looking for! We plan to make this component sticky in Q4 2019.
+
+<Row>
+<Column colMd={6} colLg={8}>
+
+![Exporting stroke app icons: step 3](/images/search.svg)
+
+</Column>
+</Row>
+
+Last updated on 9 October, 2019
+
+## Color palette v2
+
+We're pleased to announce the new IBM Design Language V2 color palette. The updates from V1 are quite subtle at first glance but incorporate significant refinements when considering contrast ratios and accessibility for digital screens and interfaces. For an in depth article outlining the methodology and process please visit [Because, colors are beautiful](https://medium.com/carbondesign/because-colors-are-beautiful-52dd4cc39f09).
+
+Major updates:
+
+- 3:1 contrast ratio between 60/90
+- 4.5:1 contrast ratio between 50/90, 40/80, 30/70, 20/60
+- slight hue enhancements to the magentas to be more distinct from the reds
+- hues across each color have been smoothed out for more consistency
+- contrast ratios have been refined for a smoother curve
+
+<Row>
+<Column colMd={6} colLg={8}>
+
+![Exporting stroke app icons: step 3](/images/color.svg)
+
+</Column>
+</Row>
+
+Last updated on 25 september 2019 <br/>
+[Find the details on our color page in IDL ](https://github.com/carbon-design-system/carbon/raw/master/packages/colors/artifacts/IBM_Colors.zip)
```

Comment:
Is this the correct text/link? Seems out of place and doesn't link to the color page.

Reply:
Updated the link to color page.